### ISSUE TITLE
What does the path of train_datasets should be?

### ISSUE THREAD

**Issue by zzy-ucas** [HuMingqi/AlexNet_tf#1]
Hello, there! Thanks for your code, but I have some problems when I running it.

In this `def create_examples(self, img_main_dir):`
> class_index= int(sub_dir.split('#')[-2]) 	# Must Append '#label_index#' to sub_dir

I do not understand what does this '#label_index#' means. Does the path of train_datasets should contain label? like " D:/train/1", " D:/train/2"...?

In this `def read_example(self, filename_queue):`

> label= tf.string_to_number(tf.string_split([id],'#').values[-2])

And I have no idea what does this means cause I am new guy to TF. 

Hope you can help me. Thanks!

**Comment by zzy-ucas** [comment]
Okay, I know 'Must Append '#label_index#' to sub_dir' should be done by me...LOL

**Comment by HuMingqi** [comment]
@zzy-ucas Thanks for your letter which lets me know my code is helpful.
And yes, It's a temporary approach by manually appending label index counted from zero. The '#' just for being convenient to get label because the poor StringOp API in TF.
If the class number are huge, you can rename all sub dirs by scripts.
